physics/src/IceGrowth.cpp

```diff
@@ -124,12 +127,14 @@ ConfigMap IceGrowth::getConfiguration() const
 
 void IceGrowth::update(const TimestepTime& tsTime)
 {
-    iFluxes->update(tsTime);
+
     // Copy the ice data from the prognostic fields to the modifiable fields.
     // Also divide by c_ice to go from cell-averaged to ice-averaged values.
     cice = cice0;
-    hice = hice0;
-    hsnow = hsnow0;
+    hice = hice0 = hIceCell / cice0;
+    hsnow = hsnow0 = hSnowCell / cice0;
```

Comment:
Shouldn't we catch the ``cice0 = 0`` case here? The suggestion below has perhaps a bit many ``=`` signs to be readable? o_O

```suggestion
    hice = hice0 = cice0 == 0. ? 0. : hIceCell / cice0;
    hsnow = hsnow0 = cice0 == 0. ? 0. : hSnowCell / cice0;
```

Reply:
A good point, but these variables are whole arrays, not scalar values, so using the plain ternary operator won't work.

Reply:
Well, I suppose it'll have to be a loop then.

Reply:
Implemented as an `overElements` loop.